libs/ledger-live-common/src/api/explorerConfig/index.ts

```diff
@@ -196,6 +196,13 @@ const initialExplorerConfig: FullConfig = {
       version: "v3",
     },
   },
+  ethereum_rinkeby: {
+    id: "eth_rinkeby",
+    stable: {
+      base: "EXPLORER_STAGING",
```

Comment:
why not "EXPLORER" 🤔  or is rinkeby only on staging today?

Reply:
Goog point too, but currently it's actually on neither :slightly_smiling_face: Backend team are working in parallel to provide it, on staging first I assume. But because of this I'll put this PR on HODL until backend ready, thanks for reminding me :+1: 

Btw Goerli just above is also still on staging while it probably was made available on prod at some point. I will check and make the change if necessary.